sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/models/notification.dto.ts

```diff
@@ -0,0 +1,9 @@
+export interface ProcessNotificationsQueueInDTO {
+  /** Maximum number of notifications to process per schedule */
+  pollingLimit: number;
+}
+
+export interface ProcessNotificationsResponseQueueOutDTO {
+  notificationsProcessed: number;
+  notificationsSuccessfullyProcessed: number;
```

Comment:
Can we add the error list here also? Even if the queue will be persisted only for 24h it can be helpful for a quick troubleshoot, if it will not add an overhead to the code.